apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.tsx

```diff
@@ -606,6 +609,7 @@ export const Confirmation: React.FC = () => {
             <PdfButton
               caseId={workingCase.id}
               title="Opna PDF þingbók og úrskurð"
+              disabled={isUpdating}
```

Comment:
Disabling the PDF button is a good idea, but does not resolve the issue reported. That had to do with the signature button.

It is possible that the issue was incorrectly reported. If not, however, then the signature button needs to be disabled as well.